settings.py

```diff
@@ -21,7 +21,6 @@ def path(*args):
 
 
 DEBUG = config('DEBUG', default=False, cast=bool)
-TEMPLATE_DEBUG = DEBUG
```

Comment:
tip: You can set `debug` in `options` if you need to keep this. https://docs.djangoproject.com/en/1.9/ref/settings/#template-debug